package_managers/install_pkgs.bzl

```diff
@@ -0,0 +1,94 @@
+#Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rule for installing apt packages from a tar file into a docker image."""
+
+def _impl(ctx):
+  builder_image_name = "bazel/%s:%s" % (ctx.attr.image_tar.label.package,
+                                        ctx.attr.image_tar.label.name.split(".tar")[0])
+  unstripped_tar = ctx.actions.declare_file(ctx.outputs.out.basename + ".unstripped")
+
+  build_contents = """\
+#!/bin/bash
+set -ex
+find .
+docker load --input {image_tar}
+
+cid=$(docker run -d -v $(pwd)/{installables}:/tmp/installables.tar -v $(pwd)/{installer}:/tmp/installer.sh --privileged {image_name} /tmp/installer.sh)
+
+docker attach $cid
+docker commit -c 'CMD /bin/bash' $cid {output_image_name}
```

Comment:
We probably want the entrypoint to be whatever it was on the base image, I wonder if we can get that from docker inspect on the base image after it's loaded?

Reply:
Entrypoint is null on the base image. Should it be null here too?